src/Compiler/test/Microsoft.NET.Sdk.Razor.SourceGenerators.Tests/RazorSourceGeneratorTests.cs

```diff
@@ -2380,6 +2380,33 @@ protected override void BuildRenderTree(global::Microsoft.AspNetCore.Components.
             Assert.Single(result.GeneratedSources);
         }
 
+        [Fact] // https://github.com/dotnet/razor/issues/7236
```

Comment:
@DustinCampbell as part of your utilities work, any interest in pulling `WorkItemAttribute` into a helper library for tests?